prompt_toolkit/key_binding/vi_state.py

```diff
@@ -9,6 +9,8 @@
 
 class InputMode(object):
     INSERT = 'vi-insert'
+    INSERT_SELECTION_LEFT = 'vi-insert-selection-left'
+    INSERT_SELECTION_RIGHT = 'vi-insert-selection-right'
```

Comment:
I'm not sure that we really need both ``INSERT_SELECTION_LEFT`` and ``INSERT_SELECTION_RIGHT``.